client/src/banners/index.tsx

```diff
@@ -58,31 +61,97 @@ function isEmbargoed(id: string) {
   }
 }
 
+function isExclusionPathname(id: string, pathname: string) {
+  if (id === MDN_PLUSPLUS_IDv1) {
+    return (
+      pathname.includes("/mdn++") ||
+      pathname.includes("/signin") ||
+      pathname.includes("/signup")
+    );
+  }
+  return false;
+}
+
+function isExclusionGeoLocation(id: string, country: string) {
+  if (id === MDN_PLUSPLUS_IDv1) {
+    return country !== "United States";
+  }
+  return false;
```

Comment:
I'd change both functions to `isIncludedX` to remove the negation, which also means we won't have to negate them where they are invoked. Double negation tends to be a tad harder to grok imo.
In this case it'd also simplify the function to:
```javascript
function isIncludedGeoLocation(id: string, country: string) {
  return id === MDN_PLUSPLUS_IDv1 && country === "United States";
}
```

Reply:
That's why code review is such a good thing. I totally can see your point and it's almost a mystery why, if I agree so much, I didn't do it myself when I was working on it.